packages/ajv-validator/tsup.config.ts

```diff
@@ -0,0 +1 @@
+export { default } from '../../tsup.config'
```

Comment:
The alternative to this file is specifying `--config ../../tsup.config.ts` in the `build` script